### ISSUE TITLE
[UPDATED]Menu map issues list.

### ISSUE THREAD

**Issue by gennariarmando** [GTAmodding/re3#470]
Issues list:

- Inconsistent min/max zoom: Initial map size and minimum max size right after zooming in it's different.

- Broken map limits over screen borders: makes you move the map far away screen borders, result in no map on screen.

- Zooming out also moves the map randomly. Edit: this is kind of wrong, map is moved accordly to mouse position, but it doesn't change the fact that it keeps moving even if the map is completely zoom out, this should be adjusted, if map is completely zoomed out, it shouldn't be possible to move it, it should stay to the center.

- Maximum zoom it's too high.

- Map needs to be filtered, you can see grid lines.

In other words and to keep it simple, reverse VC map and may add few adjustments under define.

**Comment by erorcun** [comment]
1- it's intentional
2- ok I'll fix it
3- not random, relative to mouse pos
4- no
5- I can't see grid lines, but if they're visible hiding it will be hard

most of what you listed are subjective.

**Comment by Noskillx** [comment]
Case closed imo map is perfect i see 0 issues unless im blind but i don't think so.

**Comment by gennariarmando** [comment]
I think you understood half of the issues there are, map is broken.

**Comment by erorcun** [comment]
nobody else except you thinks it's broken, this is subjective. but you're right on second one(I remember that you showed a picture of it) and I'll fix it soon.

**Comment by gennariarmando** [comment]
If you keep zooming out while the map has reached the minimum, it keeps moving randomly, that's a bug. There is more...

**Comment by erorcun** [comment]
it centers.

**Comment by ShFil119** [comment]
btw:
```
#ifdef MENU_MAP
    // VC uses -1 for coords and -2 for entities but meh, I don't want to edit DrawBlips
    if (CMenuManager::bMenuMapActive) {
        bool alreadyThere = false;
        for (int i = 0; i < NUM_MAP_LEGENDS; i++) {
            if (MapLegendList[i] == -1)
                alreadyThere = true;
        }
        if (!alreadyThere) {
            MapLegendList[MapLegendCounter] = -1;
            MapLegendCounter++;
            ArrowBlipColour1 = CRGBA(red, green, blue, alpha);
        }
    }
#endif
```
`MapLegendList` contains uint16 values

**Comment by erorcun** [comment]
> In other words and to keep it simple, reverse VC map and may add few adjustments under define.
Roger that!
Go learn how to communicate.
